pkg/controller/node/ipam/range_allocator.go

```diff
@@ -226,8 +217,16 @@ func (r *rangeAllocator) filterOutServiceRange(serviceCIDR *net.IPNet) {
 func (r *rangeAllocator) updateCIDRAllocation(data nodeAndCIDR) error {
 	var err error
 	var node *v1.Node
```

Comment:
Why does rangeAllocator need to be changed?

Reply:
Just to make the logic (and logs, verbosity, events, etc) similar across range allocator and cloud allocator. Now they're following a similar pattern.